ansible/roles/test/files/ptftests/dualtor_sniffer.py

```diff
@@ -1,57 +1,80 @@
-"""
```

Comment:
Since this file is no longer using the PTF library/framework, would it be more appropriate to move it to the `tests/scripts` directory?

Reply:
Although there is an argument to be made to keep it here, since it does not interact with the DUT directly as most of the files in `tests/scripts` do

Reply:
Agreed. Changed now.